ml-agents/mlagents/trainers/tests/tensorflow/test_simple_rl.py

```diff
@@ -291,11 +289,11 @@ def test_visual_advanced_sac(vis_encode_type, num_visual):
     _check_environment_trains(env, {BRAIN_NAME: config}, success_threshold=0.5)
 
 
-@pytest.mark.parametrize("use_discrete", [True, False])
-def test_recurrent_sac(use_discrete):
-    step_size = 0.2 if use_discrete else 0.5
+@pytest.mark.parametrize("action_sizes", [(0, 1), (1, 0)])
```

Comment:
[(0, 1), (1, 0)] is in a lot of tests, can we add (3,4) to the list to test all of these scenarios ?

Reply:
Not the tensorflow ones, right?

Reply:
I'm testing the hybrid spaces in `test_hybrid.py`